src/pharmpy/tools/structsearch/tool.py

```diff
@@ -83,6 +83,7 @@ def run_tmdd(context, model):
 
     wf = create_fit_workflow(qss_candidate_models)
     task_results = Task('results', bundle_results)
+    # FIXME : wf (Workflow) has no attribute called add_task
```

Comment:
This is correct and needs to be fixed. I won't work to use `add_task` without going via a `WorkflowBuilder`

Reply:
But since it is not related to this PR so no need to fix here.

Reply:
Will leave the comment as is for future fix then